src/app/state/transaction/saga.test.ts

```diff
@@ -7,16 +7,22 @@ import { DeepPartialRootState } from 'types/RootState'
 import { WalletErrors } from 'types/errors'
 import { transactionActions as actions } from '.'
 import { TransactionTypes } from '../paratimes/types'
-import { selectAddress, selectActiveWallet } from '../wallet/selectors'
+import { selectActiveWallet, selectAddress } from '../wallet/selectors'
 import { Wallet, WalletType } from '../wallet/types'
-import { doTransaction, submitParaTimeTransaction, getAllowanceDifference, setAllowance } from './saga'
+import { doTransaction, getAllowanceDifference, setAllowance, submitParaTimeTransaction } from './saga'
+import { addressToPublicKey } from '../../lib/helpers'
+import BleTransport from '@oasisprotocol/ionic-ledger-hw-transport-ble/lib'
 
-const makeState = (wallet: Partial<Wallet>): DeepPartialRootState => {
+const makeState = (
+  wallet: Partial<Wallet>,
+  rootState: Partial<DeepPartialRootState> = {},
```

Comment:
is DeepPartialRootState not correctly partial? :O

Reply:
Nice catch, will remove.